doc/rules-reference.md

```diff
@@ -137,6 +137,8 @@ out dependency-cruiser will assume it to be `warn`.
 A regular expression an end of a dependency should match to be catched by this
 rule.
 
+In `from`, this is the path from project root to the file containing a dependency. In `to`, this is the path from project root to the file the dependency resolves to.
```

Comment:
I wouldn't call this "project root". IMO "current working directory" matches better.

Something like: All paths are relative to the current working directory.

Reply:
The formulations supplement each other:
- technically dependency-cruiser matches from the current working directory
- typically you'll be working from the root of your project folder. Mentioning that will probably be helpful for first time users.